modules/mockserver/src/test/java/org/testcontainers/containers/MockServerContainerTest.java

```diff
@@ -14,10 +16,12 @@
 import static org.mockserver.model.HttpResponse.response;
 import static org.rnorth.visibleassertions.VisibleAssertions.assertTrue;
 
+@Slf4j
 public class MockServerContainerTest {
 
     @ClassRule
-    public static MockServerContainer mockServer = new MockServerContainer();
+    public static MockServerContainer mockServer = new MockServerContainer(MockServerClient.class.getPackage().getImplementationVersion())
```

Comment:
would be nice to test the default constructor as well (to make sure that the default version we use works fine as well)

Reply:
fixed, but I expect that it will always fail on each update since from 5.4.0 it checks for precise equality of the client and the server